### ISSUE TITLE
Add verb to install Segoe UI font

### ISSUE THREAD

**Issue by qwertychouskie** [Winetricks/winetricks#2054]
The "Segoe UI" font is required to run FATXplorer (https://fatxplorer.eaton-works.com/3-0-beta/).  It'd be nice to have a verb to install this font.

**Comment by qwertychouskie** [comment]
(I manually grabbed the TTFs from https://github.com/mrbvrz/segoe-ui-linux and it worked, just would be nice to have a verb)

**Comment by BrandowLucas** [comment]
Agreed, i just had a problem that a .exe wasn't opening and i spent days till i find out that the problem was this font (the console output didn't give me no clue whatsover), so please add this verb to winetricks.
